pkg/rancher-desktop/integrations/windowsIntegrationManager.ts

```diff
@@ -403,6 +404,29 @@ export default class WindowsIntegrationManager implements IntegrationManager {
     );
   }
 
+  protected async syncHostFile() {
+    if (this.settings.experimental?.virtualMachine?.networkingTunnel) {
```

Comment:
If this line is false should we be pulling the `hostEntry` line out of `/etc/hosts`?